app/models/team.rb

```diff
@@ -70,6 +74,18 @@ def selected
       where(kind: %w(sponsored voluntary))
     end
 
+    def by_season(year_or_season)
```

Comment:
Ok, I see. But what usecase did he intend it for?

Maybe I'm getting things wrong here, but I don't see the point in adding new _unused_ methods - that's like introducing legacy on purpose 😉